Cargo.toml

```
@@ -15,7 +15,8 @@ libc = "0.2"
 num-complex = "0.2"
 num-traits = "0.2"
 ndarray = ">=0.13"
-pyo3 = "0.10.1"
+# pyo3 = "0.10.1"
+pyo3 = { git = "https://github.com/PyO3/pyo3.git"}
```

I'm not sure if this is causing the CI failures or not, but shouldn't this be?:

```suggestion
pyo3 = "0.11.0"
```

---

Thanks, I released PyO3 0.11.1 now and will update it.